develop/devguide/Connector/ImpactOfProtocolChanges/ImpactOfProtocolVersionChanges.md

```diff
@@ -0,0 +1,103 @@
+
+- [Change primary Key](xref:ChangePrimaryKey)
+- [Change display key](xref:ChangeDisplayKey)
+- [Replace displayColumn by naming](xref:ReplaceDisplayColumnByNaming)
+- [Change to partial table](xref:ChangeToPartialTable)
+- [Change logger table](xref:ChangeLoggerTable)
+- [Change column order](xref:ChangeColumnOrder)
+- [Change displayed column order](xref:ChangeDisplayedColumnOrder)
+
+## Improving context awareness procedure
+
+A procedure has been introduced to improve context awareness and to allow certain major changes to improve the overall quality of protocols even though they could cause impact.
+
+### Context awareness
+
+Any user familiar with a product should easily find all necessary information when using DataMiner protocols. This means that DataMiner protocols should not only perform great, but should also look the part and be straightforward to use.
+
+Especially for new developments, the focus should be on getting them spot on from the start, especially since any changes that might be needed in the future could cause impact for existing users. 
+
+However, this should not stop you from improving existing protocols.
+
+### Major changes
+
+In the past, as many major changes as possible were blocked by providing workarounds or convincing the users to not make certain changes to avoid impact. While this worked, this meant new users might not get a perfect product. To improve the quality of DataMiner protocols, a different approach is therefore needed.
+
+### Procedure
+
+Whenever you receive feedback from a user that some improvements (changing parameter descriptions, change layout, etc.) could be done in a protocol, a major change can be requested to allow these changes.
+
+As a Skyline developer, you should create such requests on the [Protocol Version Request](https://intranet.skyline.be/Lists/Protocol%20Version%20Request/Overview.aspx) intranet page (similar to any other protocol version request), but no specific task is needed (unless you already have a task in your project).
+
+The *Protocol Version Request* team will evaluate the requested changes and, when the request is valid, they will create a task in the [SLC-SE-Internal Protocol Reviews](https://collaboration.skyline.be/project/5938/list) project.
+
+> [!NOTE]
+> The *Protocol Version Request* team is in charge of creating these tasks, and no tasks should be created in this project without the team knowing about it.
+
+### Tasks
+
+Whenever you are working on a task for a protocol, please keep an eye on this project and see if no other improvements could be taken along.
+
+No additional version request is needed anymore as these changes were already approved.
+
+This is also a call to all Product Owners to keep an eye on this project and try to get these tasks picked up whenever changes are done in the protocols of which they are the owner.
+
+### Workaround
+
+- [Change normalization](https://wiki.skyline.be/wiki/System_Development/Driver/Change_normalization)
```

Comment:
This link is no longer available. Can I replace this with a link to somewhere else, or should this be removed?

Reply:
I removed it